src/components/Insight/Comments.js

```diff
@@ -58,24 +63,44 @@ function editComment (id, content) {
   })
 }
 
-export default ({ id, authorId, count }) => {
+export default ({
+  id,
+  authorId,
+  count,
+  onEdit = editComment,
+  onDelete = deleteComment,
+  onCreate = createComment,
+  onGet = getComments
```

Comment:
`onEdit `, `onDelete`, ...etc are misleading function names. Function following this convention implies that function is a callback, which will be called after specific event occurred. This is clearly represented by the browser native events: `onclick`, `onkeypress`, etc. After making an according changes to the **[D-0-0]** comment, this issue will be resolved